# Data Science Tools and Ecosystem


In this notebook, Data Scince Tools and Ecosystem are summarized.


Some of the popular languages that Data Scientist use are:
1. R
2. Python
3. Java
4. SQL
5. Julia
6. C++
    

Some of the commonly used libraries used by Data Scientists include:
 1. PyTorch
 2. NumPy
 3. Matplotlib
 4. SciPy

| Data Science Tools |
| -------------------|
| JupyterLab |
| Pandas |
| Hadoop |

### Below are a few examples of evaluating aritmetic expressions in Python. 


This is a simple arithmetic expression to multiply then add integers

In [3]:
(3*4)+5

17

In [5]:
200/60

3.3333333333333335

3.3333333333333335 = 200 minutes to hours

__Objectives__: 
- Python
- C++
- R
- Java
- Julia

## Author
Nicholas Huynh